.circleci/config.yml

```diff
@@ -7240,26 +7227,6 @@ workflows:
           ios_arch: x86_64
           ios_platform: SIMULATOR
           name: pytorch_ios_12_0_0_x86_64_build
-      - pytorch_ios_build:
```

Comment:
Since iOS build jobs have been migrated to Github Action, these jobs are not running on CircleCI anyway. I assume it's fine. cc @seemethere

Reply:
We don't plan to support iOS for LTS anyways so I think this should be fine either way